libtelemetry/src/main/java/com/mapbox/android/telemetry/AttachmentMetadata.java

```diff
@@ -0,0 +1,71 @@
+package com.mapbox.android.telemetry;
+
+public class AttachmentMetadata {
+
+  private String name;
+  private String created;
+  private String fileId;
+  private String format;
+  private String type;
+  private String sessionId;
```

Comment:
Is it ok for server side that we use String as ID - if yes I am ok with this

Reply:
Confirmed that we are looking for strings server side for the IDs.